src/App.test.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+import { Route } from 'react-router-dom';
 import App from './App';
+import Header from './components/Header';
+import Home from './components/Home';
+import Shelters from './components/Shelters';
+import Hygiene from './components/Hygiene';
+import Hotlines from './components/Hotlines';
+import Food from './components/Food';
+import Transit from './components/Transit';
 
-it('renders without crashing', () => {
-  shallow(<App />);
+describe('<App />', () => {
+  const wrapper = shallow(<App />);
+  const routes = [Home, Shelters, Hygiene, Hotlines, Food, Transit];
+  it('renders properly', () => {
+    expect(wrapper.find('div').length).toBe(1);
```

Comment:
The check of the parent `div` here can be omitted, since by default you must have a parent `div` when returning more than one element. For this practice, I see no issue to check because you're verifying that it does actually exist but your component would never render without it so it's quite impossible to miss. 👍

Reply:
Although I haven't used them much, couldn't I also use a [fragment](https://reactjs.org/docs/fragments.html) instead?

Reply:
You could, yes but I don't believe `<Fragment>` is 100% fully supported so I tend to still use the old trusty `<div>'. I could be incorrect on this, but I want to say I've read where `<>` is also viable, but check and get back to me on this..

Reply:
i agree @kyleruschel, @jacobvenable by checkign that i can see you're writing tests first, once youv'e verified somthing that would require that first test to be true you can remove it in a refactor....`Red`, `Green`, `Refactor` 👍  👏